src/sass/components/_button.scss

```diff
@@ -5,12 +5,14 @@
   line-height: 24px;
   font-weight: 600;
   padding: 10px 30px;
+}
 
-  &:not(.no-hover):hover {
-    color: #ffffff !important;
-    background-color: $primary;
-    text-decoration: none;
-  }
```

Comment:
Dlaczego został usunięty efekt hover na przyciskach?

Reply:
bo tak zrozumialam polecenie. ze dwa przyciski maja sie pojawiac w momencie najechania na box produktu, a na 3cim ma pojawiac sie zmiana koloru, rowniez po najechaniu na box-produkt

Reply:
Tylko, że efekt hover został zabrany przez to dla wszystkich przycisków, a nie tylko dla box produktu